images/runtime/download-cni.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Copyright 2017-2020 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+set -o xtrace
+set -o errexit
+set -o pipefail
+set -o nounset
+
+source /tmp/cni-version.sh
+
+for arch in amd64 arm64 ; do
+  curl --fail --show-error --silent --location "https://github.com/containernetworking/plugins/releases/download/v${cni_version}/cni-plugins-linux-${arch}-v${cni_version}.tgz" --output "/tmp/cni-${arch}.tgz"
```

Comment:
Should we add retries here in case there is a temporary connectivity failure?

Reply:
Perhaps, but I'd rather leave that for another PR, and ideally we should update all calls to curl in `cilium/image-tools` too.